test/db/cmd/cmd_print

```diff
@@ -744,8 +744,8 @@ RUN
 NAME=prc text color
 FILE=-
 CMDS=<<EOF
-. bins/other/palette.r2
-.. bins/src/prc_256.py
+. scripts/palette.r2
+.. scripts/0-to-0xff.py
```

Comment:
wtf? why not just use woe

Reply:
oh sorry d11f441e29a9b02f11f237037782aa04fb48dc33